keyboards/kbdcraft/adam64/info.json

```diff
@@ -0,0 +1,220 @@
+{
+
+    "keyboard_name": "Adam64",
+    "manufacturer": "KBDCRAFT",
+    "url": "https://kbdcraft.store/products/adam",
+    "maintainer": "KBDCraft",
+    "usb": {
+        "vid": "0x586A",
+        "pid": "0xB001",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "rows": ["C8", "C9", "A8", "A10", "B14"],
+        "cols": ["B15", "C6", "C7", "A15", "B7", "C10", "C11", "C12", "D2", "B3", "B4", "B5", "A0", "A1"]
+    },
+    "features":{
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": true,
+        "nkro": true,
+        "console": false,
+        "command": false,
+        "backlight": false,
+        "rgblight": false,
+        "audio": false,
+        "lto": false,
+        "dip_switch": false,
+        "rgb_matrix": true
+    },
+    "rgb_matrix":{
+        "driver": "IS31FL3741",
+        "max_brightness": 150,
+        "val_steps": 50,
```

Comment:
50 seems like a rather large value, and using the default is preferred 
```suggestion
```

Reply:
Removed `"val_steps": 50,` from `info.json`